### PR TITLE
Appraiser moderation rules changed, review acceptance with comments added

### PR DESCRIPTION
**Pull request by Space-tourist**
Appraiser moderation rules changed, review acceptance with comments added

### CODE REVIEW

File: src/Inc.TeamAssistant.Reviewer.Domain/TaskForReview.cs [hunk 1]
```diff
@@ -78,17 +79,29 @@ public void SetNextNotificationTime(DateTimeOffset now, NotificationIntervals no
...
+            MoveToAcceptWithComments();
+            return;
+        }
+
         MoveToAccept();
     }
     
     public void MoveToAccept() => State = TaskForReviewState.Accept;
 
+    public void MoveToAcceptWithComments()
```
Comment (dyatlov-a) [inline src/Inc.TeamAssistant.Reviewer.Domain/TaskForReview.cs:99]: Mark this method as private.

File: src/Inc.TeamAssistant.Reviewer.Application/CommandHandlers/MoveToAccept/MoveToAcceptCommandHandler.cs [hunk 2]
```diff
@@ -42,7 +42,7 @@ public async Task<CommandResult> Handle(MoveToAcceptCommand command, Cancellatio
         if (owner is null)
             throw new TeamAssistantUserException(Messages.Connector_PersonNotFound, taskForReview.OwnerId);
 
-        taskForReview.Accept(DateTimeOffset.UtcNow);
+        taskForReview.Accept(DateTimeOffset.UtcNow, command.AcceptedWithComments);
```
Comment (dyatlov-a) [inline src/Inc.TeamAssistant.Reviewer.Application/CommandHandlers/MoveToAccept/MoveToAcceptCommandHandler.cs:45]: Please create a separate commit for each task. A note for the future.

File: src/Inc.TeamAssistant.Reviewer.Domain/TaskForReview.cs (hunk 1 above)
Comment (Space-tourist) [reply src/Inc.TeamAssistant.Reviewer.Domain/TaskForReview.cs]: fixed

File: src/Inc.TeamAssistant.Reviewer.Application/CommandHandlers/MoveToAccept/MoveToAcceptCommandHandler.cs (hunk 2 above)
Comment (Space-tourist) [reply src/Inc.TeamAssistant.Reviewer.Application/CommandHandlers/MoveToAccept/MoveToAcceptCommandHandler.cs:45]: ok

File: src/Inc.TeamAssistant.WebUI/Features/Constructor/Stages/BotPropertiesBuilder.cs [hunk 3]
```diff
@@ -9,7 +9,8 @@ public static class BotPropertiesBuilder
         ["nextReviewerStrategy"] = "RoundRobin",
         ["waitingNotificationInterval"] = "00:30:00",
         ["inProgressNotificationInterval"] = "01:00:00",
-        ["storyType"] = "Fibonacci"
+        ["storyType"] = "Fibonacci",
+        ["acceptWithComments"] = "No"
```
Comment (dyatlov-a) [inline src/Inc.TeamAssistant.WebUI/Features/Constructor/Stages/BotPropertiesBuilder.cs:13]: Set default - Yes

File: src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs [hunk 4]
```diff
@@ -40,6 +41,7 @@ public async Task<IReadOnlyCollection<NotificationMessage>> Build(
         TaskForReview task,
         Person reviewer,
         Person owner,
+        BotContext botContext,
```
Comment (dyatlov-a) [inline src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs:44]: Please add check for null

File: src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs [hunk 5]
```diff
@@ -264,6 +266,7 @@ private async Task<NotificationMessage> HideControlsForOriginalReviewer(
 
     private async Task<NotificationMessage> MessageForReviewer(
         TaskForReview task,
+        BotContext botContext,
```
Comment (dyatlov-a) [inline src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs:269]: Please add check for null

File: src/Inc.TeamAssistant.WebUI/Features/Constructor/Stages/BotPropertiesBuilder.cs (hunk 3 above)
Comment (Space-tourist) [reply src/Inc.TeamAssistant.WebUI/Features/Constructor/Stages/BotPropertiesBuilder.cs]: fixed

File: src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs (hunk 4 above)
Comment (Space-tourist) [reply src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs:44]: fixed

File: src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs (hunk 5 above)
Comment (Space-tourist) [reply src/Inc.TeamAssistant.Reviewer.Application/Services/ReviewMessageBuilder.cs:270]: fixed

**Review by dyatlov-a — APPROVED** [review]

**Pull request merged**